src/aluno/PerfilPage.tsx

```
@@ -139,60 +174,133 @@ export function PerfilPage() {
-                                        <p className="text-sm text-gray-500">Você ainda não participa de nenhum projeto.</p>
-                                    ) : (
-                                        <ul className="space-y-2">
-                                            {projetosUsuario.map((projeto) => (
-                                                <li key={projeto.id}>
-                                                    <button
-                                                        className="w-full text-left p-2 !border !bg-transparent !border-gray-200"
-                                                        onClick={() => {
-                                                            setSelectedProjeto(projeto)
-                                                            setDialogOpen(true)
+                            <CardContent className="space-y-2 !p-0">
+                                {projetosUsuario.length === 0 ? (
```

ESLint

[error] Delete `················`

(prettier/prettier)